test/unit/rules/no-nested-landmark-test.js

```diff
@@ -13,6 +13,10 @@ generateRuleTests({
   good: [
     '<div><main></main></div>',
     '<div role="application"><div role="document"><div role="application"></div></div></div>',
+    '<footer><nav></nav></footer>', // nested landmarks of different types are okay
```

Comment:
While this is technically okay, it's not great UX. You'd also need to make sure that a nav in a footer has some sort of accessible name, because if the nav is in the footer there is likely another nav somewhere on the page.

Reply:
@MelSumner are you saying that nested landmarks in general are not great UX, or are you saying that a nav nested in a footer is not great UX?  If it's the latter, I'll be happy to change the passing examples in the test suite.  If it's the former then I should probably just close the issue and the PR.